Overview.bs

```diff
@@ -534,20 +534,20 @@ Media: interactive
 Animatable: no
 </pre>
 
-<dl dfn-type=value dfn-for=animator-root>
+<dl dfn-type=value dfn-for=animator>
```

Comment:
nit: I think I'd prefer to define animator-root first so that when animator refers to roots it is already established what an animator-root is.